cluster/up.sh

```diff
@@ -21,37 +21,25 @@ kubevirtci::install
 
 $(kubevirtci::path)/cluster-up/up.sh
 
-# Deploy CNA
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/namespace.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/network-addons-config.crd.yaml
-./cluster/kubectl.sh create -f https://github.com/kubevirt/cluster-network-addons-operator/releases/download/0.15.0/operator.yaml
-./cluster/kubectl.sh create -f ./hack/cna/cna-cr.yaml
-
-# wait for cluster operator
+echo 'Deploying Linux bridge CNI and Multus ...'
```

Comment:
why do we need multus and linux bridge deployed? is it so we have "customers" to give MACs to?

Reply:
Yes. In e2e tests we connect Pods and VMs to secondary networks via Multus and the bridge CNI.